cmd/keytransparency-client/cmd/post.go

```diff
@@ -74,10 +72,8 @@ User email MUST match the OAuth account used to authorize the update.
 		if err != nil {
 			return fmt.Errorf("error connecting: %v", err)
 		}
-		ctx, cancel := context.WithTimeout(context.Background(), timeout)
+		cctx, cancel := context.WithTimeout(ctx, timeout)
```

Comment:
Can possibly just reuse ctx here or pass context.Background(). Actually might want to use context.TODO here?

Reply:
This is the main function, so context.Background feels appropriate. 
Not sure about reusing ctx. I like to keep cctx specific to a function, and to make it clear that this cctx will timeout very soon.